### PR TITLE
feat(pmp-web): single user statistic data access

### PR DESCRIPTION
**Pull request by Shaelex**
Needed to #17

### CODE REVIEW

File: libs/pmp-web/user/single-user-statistics/data-access/src/lib/+state/single-user-statistics.facade.ts [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+export class SingleUserStatisticsFacade {
+  singleUserStatisticsResponse$ = this.store.pipe(
+    select(singleUserStatisticsQuery.getSingleUserStatisticsResponse)
+  );
+  singleUserStatisticsResponseLoading$ = this.store.pipe(
+    select(singleUserStatisticsQuery.getSingleUserStatisticsResponseLoading)
+  );
+  singleUserStatisticsResponseLoadError$ = this.store.pipe(
+    select(singleUserStatisticsQuery.getSingleUserStatisticsResponseLoadError)
+  );
```
Comment (Stefaneek) [inline libs/pmp-web/user/single-user-statistics/data-access/src/lib/+state/single-user-statistics.facade.ts]: Add one line space

File: libs/pmp-web/user/single-user-statistics/feature/src/lib/containers/single-user-statistics/single-user-statistics.component.ts [hunk 2]
```diff
@@ -1,54 +1,52 @@
...
 @Component({
   selector: 'pmp-single-user-statistics',
   templateUrl: './single-user-statistics.component.html',
   styleUrls: ['./single-user-statistics.component.scss'],
   changeDetection: ChangeDetectionStrategy.OnPush
 })
-export class SingleUserStatisticsComponent implements OnInit {
-  mockedData = mockedData;
+export class SingleUserStatisticsComponent implements OnInit, OnDestroy {
   user: string | null;
```
Comment (Stefaneek) [inline libs/pmp-web/user/single-user-statistics/feature/src/lib/containers/single-user-statistics/single-user-statistics.component.ts]: Maybe name it as userName?

File: libs/pmp-web/user/single-user-statistics/data-access/src/lib/services/single-user-statistics-data.service.ts [hunk 3]
```diff
@@ -0,0 +1,31 @@
+import { Injectable } from '@angular/core';
+import { Observable } from 'rxjs';
+import { HttpClient } from '@angular/common/http';
+import { GetSingleUserStatisticsResponseRequestPayload } from '../resources/request-payloads/get-single-user-statistics-response.request-payload';
+import { IResponse, SingleUserStatisticsResponse } from '@pimp-my-pr/shared/domain';
+import { urlFactory } from '@valueadd/typed-urls';
+import { map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
```
Comment (wjanaszek) [inline libs/pmp-web/user/single-user-statistics/data-access/src/lib/services/single-user-statistics-data.service.ts]: Provide it in module

Comment (Stefaneek) [inline libs/pmp-web/user/single-user-statistics/data-access/src/lib/services/single-user-statistics-data.service.ts]: Should be provided in the module

**Pull request merged**